src/d/actor/d_a_obj_table.cpp

```diff
@@ -4,78 +4,185 @@
 //
 
 #include "d/actor/d_a_obj_table.h"
+#include "d/d_a_obj.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
+#include "f_op/f_op_actor_mng.h"
+
+Mtx daObjTable::Act_c::M_tmp_mtx;
+const char daObjTable::Act_c::M_arcname[6] = "Table";
 
 /* 00000078-000001B8       .text CreateHeap__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::CreateHeap() {
-    /* Nonmatching */
+    u8 prm;
+    J3DModelData *model_data;
+    J3DModel *model;
+
+
+    prm = daObj::PrmAbstract(this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+    if (!prm) {
```

Comment:
If you check the debug map, you'll see that this actor has an inline:
```suggestion
    if (prm_get_mdl() == 0) {
```
There's an empty placeholder for it in the actor's header, but you need to implement its body.

And you can also change the enum name "LINK_ID" which were copy pasted from a different actor to instead say "MDL" now that we know the real name of this parameter.